testing/template/template.txt

```diff
@@ -0,0 +1,30 @@
+%% md
```

Comment:
We won't need this file because you will create it dynamically in the analysis step.